pegen/c_generator.py

```diff
@@ -217,7 +217,12 @@ def out_of_memory_goto(
 
     def generate(self, filename: str) -> None:
         self.collect_todo()
-        self.print(EXTENSION_PREFIX.format(filename=filename))
```

Comment:
Are you sure you want to allow overriding the `// @generated by pegen.py from {filename}` line?